### PR TITLE
Remove year from date of birth field

### PR DESCRIPTION
**Pull request by erquirogasw**
## Description
This PR updates the _DateOfBirth_ field format to "MM-dd" in order to not use the year and follow good data privacy practices.

## Specific Changes
  - Updated format of _Birthday_ property to "MM-dd".

The following image shows the code changes and the data result.
image

### CODE REVIEW

File: azure-functions/DataIngestionBambooAPI/DataIngestionBambooAPI.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public static class DataIngestionBambooAPI
         static Config config;
 
         [FunctionName("DataIngestionBambooAPI")]
-        public static async Task Run([TimerTrigger("0 0 9 * * *")] TimerInfo myTimer, ILogger log, ExecutionContext context)
+        public static async Task Run([TimerTrigger("0 0 9 * * *", RunOnStartup = false)] TimerInfo myTimer, ILogger log, ExecutionContext context)
```
Comment (JuanAr) [inline azure-functions/DataIngestionBambooAPI/DataIngestionBambooAPI.cs:12]: This one goes to another PR

**Review by JuanAr — CHANGES_REQUESTED** [review]
Split in two PRs one per each topic

File: azure-functions/DataIngestionBambooAPI/DataIngestionBambooAPI.cs (hunk 1 above)
Comment (erquirogasw) [reply azure-functions/DataIngestionBambooAPI/DataIngestionBambooAPI.cs]: Done. Thanks!

**Review by cristian-uehara — APPROVED** [review]
lgtm!

**Review by cristian-uehara — APPROVED** [review]
lgtm

**Review by JuanAr — APPROVED** [review]
LGTM

**Pull request merged**